### ISSUE TITLE
MP3 playback terminates before all audio has been output

### ISSUE THREAD

**Issue by tucson-tom** [popcornmix/omxplayer#641]
Sample attached.  The 's' at the end does not play via omxplayer.  VLC on my desktop works fine.

[last_pushed_news_speech.zip](https://github.com/popcornmix/omxplayer/files/2182474/last_pushed_news_speech.zip)

```
# omxplayer --version
omxplayer - Commandline multimedia player for the Raspberry Pi
        Build date: Fri, 23 Sep 2016 00:56:49 +0000
        Version   : dfea8c9 [master]
        Repository: https://github.com/popcornmix/omxplayer.git
```

**Comment by mfxuus** [comment]
Hey I know this is an old issue - just wondering if you ever got this resolved?

**Comment by tucson-tom** [comment]
Nope.  Never got it resolved and eventually stopped running the project that this was associated with.

**Comment by popcornmix** [comment]
omxplayer has been deprecated in favour of vlc.
If you want to play mp3 files there are many alternatives that are designed for audio.

**Comment by mfxuus** [comment]
Thanks for the responses folks! For my particular situation I don't have a lot of flexibility in terms of choosing other alternatives, or for that matter, even updating the omxplayer version would be difficult. I was really weird since it's working properly for some but not others, and comparing the logs (which I'm no expert in reading them) they seem to be very similar / nothing stands out.
